cdt_slugs.yaml

```diff
@@ -597,6 +603,24 @@ webkitgtk-devel:
     - cos7-aarch64
     - cos7-ppc64le
   license_file: licenses/webkitgtk-license
+xcb-util-devel:
+  custom: false
+  license_file: licenses/xcb-util-license
+xcb-util-image-devel:
+  custom: false
+  license_file: licenses/xcb-util-image-license
+xcb-util-keysyms:
+  custom: false
+  license_file: licenses/xcb-util-keysyms-license
+xcb-util-keysyms-devel:
+  custom: false
+  license_file: licenses/xcb-util-keysyms-license
```

Comment:
@isuruf, the CI is complaining about the license of `xcb-util-keysyms`, however it is listed here. Am I doing something wrong here?